keyboards/fc980c/ad5258.c

```diff
@@ -24,7 +24,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 // AD5258 I2C digital potentiometer
 // http://www.analog.com/media/en/technical-documentation/data-sheets/AD5258.pdf
 //
-#define AD5258_I2C_ADDRESS 0x18
+#define AD5258_I2C_ADDRESS (0x18<<1) // i2c_master API requires shifting address one bit to the left
```

Comment:
```suggestion
#define AD5258_I2C_ADDRESS 0x30
```

Reply:
Hmm, I can drop the comment, but I think explicitly having the shift reduces confusion since `0x18` is the address actually documented in the datasheet.